graylog2-web-interface/src/logic/notifications/NotificationsFactory.jsx

```diff
@@ -14,406 +14,32 @@
-              For this reason Elasticsearch enforces a read-only index block on all indexes having any of their shards in any of the{' '}
-              affected nodes. The affected nodes are: [{notification.details.nodes}]{' '}
-              Check <a href="https://www.elastic.co/guide/en/elasticsearch/reference/master/disk-allocator.html" target="_blank" rel="noreferrer">https://www.elastic.co/guide/en/elasticsearch/reference/master/disk-allocator.html</a>{' '}
-              for more details.
-            </span>
-          ),
-        };
-
-      case 'es_version_mismatch': {
-        const { initial_version: initialVersion, current_version: currentVersion } = notification.details;
-
-        return {
-          title: 'Elasticsearch version is incompatible',
-          description: (
-            <span>
-              The Elasticsearch version which is currently running ({currentVersion}) has a different major version than
-              the one the Graylog leader node was started with ({initialVersion}).{' '}
-              This will most probably result in errors during indexing or searching. Graylog requires a full restart after an
-              Elasticsearch upgrade from one major version to another.
-              <br />
-              For details, please see our notes about{' '}
-              <DocumentationLink page={DocsHelper.PAGES.ROLLING_ES_UPGRADE}
-                                 text="rolling Elasticsearch upgrades." />
-
-            </span>
-          ),
-        };
-      }
-
       case 'legacy_ldap_config_migration': {
         const { auth_service_id: authServiceId } = notification.details;
-        const authServiceLink = <Link to={Routes.SYSTEM.AUTHENTICATION.BACKENDS.show(authServiceId)}>Authentication Service</Link>;
 
         return {
-          title: 'Legacy LDAP/Active Directory configuration has been migrated to an Authentication Service',
-          description: (
-            <span>
-              The legacy LDAP/Active Directory configuration of this system has been upgraded to a new {authServiceLink}.
-              Since the new {authServiceLink} requires some information that is not present in the legacy
-              configuration, the {authServiceLink} <strong>requires a manual review</strong>!
-              <br />
-              <br />
-              <strong>After reviewing the {authServiceLink} it must be enabled to allow LDAP or Active Directory users
-                to log in again!
-              </strong>
-              <br />
-              <br />
-              Please check the <DocumentationLink page={DocsHelper.PAGES.UPGRADE_GUIDE} text="upgrade guide" />
-              for more details.
-            </span>
-          ),
+          values: {
+            AUTHENTICATION_BACKEND: Routes.SYSTEM.AUTHENTICATION.BACKENDS.show(authServiceId),
+          },
         };
       }
 
-      case 'archiving_summary':
+      case 'no_input_running': {
```

Comment:
```suggestion
      case 'no_input_running':
      case 'input_failure_shutdown':
      case 'input_failed_to_start': {
```

@ousmaneo there are a few missing here